Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM centos:7.4.1708
```

Comment:
@StevenTobin @darahayes Should the base image for the api server also be rhscl/rhel-based?

Reply:
@paolobueno I think for now this should be okay. This is all upstream so it's not the biggest deal.

Reply:
The centos image is the upstream image so I think it's fine to have here :+1: